docs/usage/shoot-extension.md

```diff
@@ -10,22 +10,59 @@ In most of the Gardener setups the `shoot-lakom-service` extension is enabled gl
 kind: Shoot
 ...
 spec:
+  resources:
+  - name: lakom-ref
+    resourceRef:
+      apiVersion: v1
+      kind: Secret
+      name: lakom-secret
   extensions:
   - type: shoot-lakom-service
     disabled: true
     providerConfig:
       apiVersion: lakom.extensions.gardener.cloud/v1alpha1
       kind: LakomConfig
       scope: KubeSystem
+      publicKeysSecretReference: lakom-ref
 ...
 ```
 
+### Scope
+
 The `scope` field instruct lakom which pods to validate. The possible values are:
 
 - `KubeSystem`
 Lakom will validate all pods in the `kube-system` namespace.
 - `KubeSystemManagedByGardener`
-Lakom will validate all pods in the `kube-system` namespace that are annotated with "managed-by/gardener"
+Lakom will validate all pods in the `kube-system` namespace that are annotated with "managed-by/gardener". This is the default value.
 - `Cluster`
 Lakom will validate all pods in all namespaces.
 
+### PublicKeysSecretReference
+
+Lakom, by default, tries to verify only workloads that belong to Gardener. Because of this, the only public keys that it uses to do its job are the ones for the Gardener workload.
+
+If you'd like to use Lakom as a tool for verifying your own workload, you'll need to add your own public keys to the ones that Lakom is already using. This can be achieved using Gardener [referenced resources](https://github.com/gardener/gardener/blob/master/docs/extensions/referenced-resources.md).
+
+Simply:
+1. Create a secret in your project namespace that contains a field `keys` with your keys as a value. Example keys:
```

Comment:
It would be nice to document the format of a key (`name`, `algorithm` and `key` fields). Or for simplicity, add a reference to https://github.com/gardener/gardener-extension-shoot-lakom-service/blob/main/docs/usage/lakom.md#lakom-cosign-public-keys-configuration-file